examples/game_of_life_2D_CA.jl

```diff
@@ -90,17 +89,15 @@ end
 
 # ## 3. Animate the model
 
-# We use the `plotabm` function from `AgentsPlots.jl` package for creating an animation.
+# We use the `abm_video` function from `InteractiveChaos.jl` package for creating an animation and saving it to an mp4
 
 cd(@__DIR__) #src
+using InteractiveChaos, CairoMakie
 ac(x) = x.status == true ? :black : :white
-anim = @animate for i in 0:100
-    i > 0 && step!(model, dummystep, ca_step!, 1)
-    p1 = plotabm(model; ac = ac, as = 3, am = :square, showaxis = false)
-end
+am(x) = x.status == true ? '■' : '□'
+abm_video(
+    "game of life.mp4", model, dummystep, ca_step!;
+    title="Game of Life", ac=:black, as=12,
+    am = am, framerate=5
+)
 nothing # hide
-
```

Comment:
If you take a look at some of the files I've updated, after the `abm_video` call we need to put
```julia
nothing # hide
# ```@raw html
# <video width="auto" controls autoplay loop>
# <source src="../file_name.mp4" type="video/mp4">
# </video>
# ```
```

Additionally, under the first heading in most examples, there's a link to the gif file. That can be replaced by the raw html block as well.